packages/flutter_tools/lib/src/compile.dart

```diff
@@ -656,6 +656,7 @@ class DefaultResidentCompiler implements ResidentCompiler {
       sdkRoot,
       '--incremental',
       '--target=$targetModel',
+      '--debugger-module-names',
```

Comment:
Are there any comments worth adding here about what this flag does?

Reply:
Added a comment. This is a breaking change flag, so once all of the tooling is updated we can make it the default and remove

Reply:
Is there an issue we can link to for the migration?

Reply:
Filled and linked https://github.com/flutter/flutter/issues/52693